impeller/geometry/geometry_benchmarks.cc

```diff
@@ -82,11 +157,36 @@ static void BM_Convex(benchmark::State& state, Args&&... args) {
   state.counters["TotalPointCount"] = point_count;
 }
 
+#define MAKE_STROKE_BENCHMARK_CAPTURE(path, cap, join)                 \
+  BENCHMARK_CAPTURE(BM_StrokePolyline, stroke_##path##_##cap##_##join, \
+                    Create##path(), Cap::k##cap, Join::k##join, false)
+
+#define MAKE_STROKE_BENCHMARK_CAPTURE_UV(path, cap, join)                   \
+  BENCHMARK_CAPTURE(BM_StrokePolyline, stroke_##path##_##cap##_##join##_uv, \
+                    Create##path(), Cap::k##cap, Join::k##join, true)
+
+#define MAKE_STROKE_BENCHMARK_CAPTURE_CAPS_JOINS(path, uv) \
+  MAKE_STROKE_BENCHMARK_CAPTURE##uv(path, Butt, Bevel);    \
+  MAKE_STROKE_BENCHMARK_CAPTURE##uv(path, Butt, Miter);    \
+  MAKE_STROKE_BENCHMARK_CAPTURE##uv(path, Butt, Round);    \
+  MAKE_STROKE_BENCHMARK_CAPTURE##uv(path, Square, Bevel);  \
```

Comment:
Why leave off combos like Square + Miter?

Reply:
I could add them. I just thought that we should test one attribute at a time, as I describe in the "notes" comment above.

Reply:
Oh, yeah - makes sense. I don't think we need the full combinatorial set.

Reply:
Though all combinations are tested for both uv and non-uv. I could go either way on filling out the rest of the cap/join combinations. Note that it would take use from 5 to 9 variations * 2 path types * UV/nonUV